uploader.html

```diff
@@ -283,7 +272,7 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
         progressBarSelector: "#js-file-progress",
         readyTextSelector: "#js-file-ready-text",
       });
-      mediaUploader.init();
+      mediaUploader.init('image');
```

Comment:
Curious why this `init(...)` call is separate from the constructor? Is it intended to be called multiple times per `MultimediaUploader` instance? If so, I'd suggest renaming it to something like `addFileType(type)`. If not, I'd suggest putting the filetype(s) in a property of the options passed into the constructor and make the `init` code private/called by the constructor.

Reply:
Not entirely sure I understand this comment.  Can you provide a little more description of the desired change?

Reply:
https://github.com/knittingarch/MultimediaUploader/pull/1/files/bb8bf535f1146b52a7b1ebdb1552ec649e140ac4#r140398360